src/module/skill-actions-data.ts

```diff
@@ -0,0 +1,74 @@
+import { SkillAction } from './skill-actions';
+
+const skillActionsData = [
```

Comment:
I kind of wanted to move this into compendium, but a module cannot define new item types and existing item types won't work for us since we have custom data. Also it would be weird to have compendium of items that can't be added to actor.

My other option was some kind of JSON file, but I also can't figure out how to use those in foundry.

That leaves this, which is OK I guess.